src/constants.ts

```diff
@@ -0,0 +1,4 @@
+import { getNumbers } from './utils';
```

Comment:
It seems that the `getNumbers` function is being imported but not used anywhere within this file. If this function is not required, consider removing the import statement to keep the code clean and maintainable.